content/solana-nft.md

```diff
@@ -0,0 +1,618 @@
+# Creating Solana NFTs With Metaplex
+
+# Lesson Objectives
+
+*By the end of this lesson, you will be able to:*
+
+- Understand the role of Metaplex in the Solana NFT ecosystem
+- Use Candy Machine v2 to create an NFT collection
+- Use a frontend UI to mint NFTs from a Candy Machine
+
+# TL;DR
+
+- **Non-Fungible Tokens (NFTs)** are represented on Solana as SPL Tokens with an associated metadata account, 0 decimals, and a maximum supply of 1
+- **Metaplex** is a collection of tools that simplify the creation and distribution of NFTs on the Solana blockchain
+- **Candy Machine v2** is an NFT distribution program by Metaplex used to mint NFTs from a collection
+
+# Overview
+
+Solana Non-Fungible Tokens (NFTs) are SPL tokens created using the Token Program. These tokens, however, also have an additional metadata account associated with each token mint. This allows for a wide variety of use cases for tokens. You can effectively tokenize anything, from game inventory to art. 
+
+While a deep dive on the use cases for NFTs would be fascinating, this lesson will focus on the creation and distribution of NFTs.
+
+In this lesson, we'll cover the basics of how NFTs are represented on Solana, as well as how to create, distribute, and update a new NFT collection.
+
+## NFTs on Solana
+
+A Solana NFT is a non-divisible token with associated metadata that comes from a token mint with a maximum supply of 1. Let's break that down a bit. An NFT is a standard token from the Token Program. What makes it unique is that it:
+
+1. Has 0 decimals so that it cannot be divided into parts
+2. Comes from a token mint with supply of 1 so that only 1 of these tokens exists
+3. Comes from a token mint whose authority is set to null (to ensure that the supply never changes)
+4. Has an associated account that stores metadata
+
+While the first three points are standard features that can be achieved with the SPL Token Program, the associated metadata requires some additional functionality.
+
+Typically, an NFT’s metadata has both an on-chain and off-chain component. The on-chain metadata is stored in an account associated with the token mint and contains a URI attribute that points to an off-chain JSON file. The off-chain component stores additional data and a link to the image. Permanent data storage systems such as Arweave are often used to store the off-chain component of NFT metadata.
+
+Below is an example of the relationship between on-chain and off-chain metadata. The on-chain metadata contains a URI field that points to an off-chain `.json` file that stores the link to the image of the NFT and additional metadata.
+![Screenshot of Metadata](../assets/solana-nft-metaplex-metadata.png)
```

Comment:
@jamesrp13 newline separation between line 38 and the image?

Reply:
@jamesrp13 bump

Reply:
K I expected it to show up in the conversation but it didn't. Apparently you need to have this as part of a review for it to show up. Single comments don't hack it I guess

Reply:
But will fix the separation thanks